src/raster/rasterband.rs

```diff
@@ -1362,8 +1357,8 @@ impl ColorInterpretation {
 
     /// Returns the name of this color interpretation.
     pub fn name(&self) -> String {
-        let rv = unsafe { gdal_sys::GDALGetColorInterpretationName(self.c_int()) };
-        _string(rv)
+        let c_ptr = unsafe { gdal_sys::GDALGetColorInterpretationName(self.c_int()) };
```

Comment:
`c_ptr` is owned by `self` so we don't need to free it?

Reply:
`GDALGetColorInterpretationName` returns static strings.